docs/select.rst

```diff
@@ -289,6 +291,20 @@ Filters on links
   }));
 
 
+You can also use the :ref:`type intersection
+<edgedb-js-objects-type-intersections>` operator to filter a link based on its
+type.
```

Comment:
Do you think we can add some more detail about when and why you might need to do this? I think the docs in the Standard Library do a decent job here, but it might be helpful to relate it to _this_ specific schema. Maybe something like:

```suggestion
You can also use the :ref:`type intersection
<edgedb-js-objects-type-intersections>` operator to filter a link based on its
type. For example, since `actor.roles` might be of type `Movie` or `TVShow`,
to only return `roles` that are `Movie`s, you would use the `.is` type
intersection operator:
```